src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -24,9 +25,10 @@
    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
     //dummy data for user
-    String username = "Sam";
-    
+    String authenticatedEmail = "sam@gmail.com";
+
     Gson gson = new Gson();
 
     Query query = new Query("User");
```

Comment:
This query can also be initialised as a class variable as it doesn't change from query to query.